### PR TITLE
feat: add bakosfe connector

### PR DESCRIPTION
**Pull request by guimroque**
- add bakosafe on default connectors
- fix visual bug on scroll

### CODE REVIEW

File: packages/bako-safe/src/DAPPWindow.ts [hunk 1]
```diff
@@ -73,6 +75,9 @@ export class DAppWindow {
   }
 
   makeFrame(method: string, isSafari = false) {
+    //go to top
+    document.scrollingElement?.scrollTo(0, 0);
```
Comment (luizstacio) [inline packages/bako-safe/src/DAPPWindow.ts:79]: Instead of scrolling to the top, you can fix the container on the screen size.

**Review by luizstacio — CHANGES_REQUESTED** [review]

File: packages/bako-safe/src/DAPPWindow.ts (hunk 1 above)
Comment (guimroque) [reply packages/bako-safe/src/DAPPWindow.ts:79]: this fix is ​​just to resolve this bug:


Uploading RPReplay_Final1718319302.mov…



- when the user is at the end of the page, even if you pass 100% of the screen height, we will have this problem.

File: .changeset/forty-timers-heal.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-connectors/bako-safe": major
```
Comment (LuizAsFight) [inline .changeset/forty-timers-heal.md:2]: not a major. it's a patch

File: .changeset/forty-timers-heal.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+Add bako-safe connector
```
Comment (LuizAsFight) [inline .changeset/forty-timers-heal.md:5]: the bako-safe connector changeset text should be something like:

```

fix positioning/size of bako popup
```

**Review by LuizAsFight — CHANGES_REQUESTED** [review]

File: .changeset/forty-timers-heal.md (hunk 3 above)
Comment (LuizAsFight) [reply .changeset/forty-timers-heal.md:5]: and we need another patch changeset for the package `@fuels/connectors`:

```
add bako safe connector to the default connectors
```

**Review by LuizAsFight — APPROVED** [review]

**Review by luizstacio — APPROVED** [review]

**Pull request merged**